test/graph/identity_wrapper.go

```diff
@@ -29,14 +29,17 @@ func loadIdentityWrapper(g *TestGraph, identityID uuid.UUID) identityWrapper {
 func newIdentityWrapper(g *TestGraph, params []interface{}) interface{} {
 	w := identityWrapper{baseWrapper: baseWrapper{g}}
 	var lastActive *time.Time
+	username := "TestUserIdentity-" + uuid.NewV4().String()
 	for _, p := range params {
 		switch p := p.(type) {
+		case string:
```

Comment:
Make sure you test for the equivalent pointer types also.

Reply:
ok, let me add that

Reply:
done in ff39a85